### ISSUE TITLE
Latest version of KeePassHTTP does not work with latest keepass

### ISSUE THREAD

**Issue by clauderobi** [pfn/keepasshttp#389]
The latest version of KeePasshttp does not work with Keepass 2.41

Can you update it?

**Comment by mvenghaus** [comment]
same here

**Comment by tps800** [comment]
It works, but wont display any dialog if it finds more than one login for one URL.
Workaround: open webpage to login. Webpage seems to load, but than does not display anything, loading forever. No login-dialog will be shown. If you click on the keepasshttp icon, you'll have items to select from to login. Select one. You'll be logged in then and the page continues to load.

Seems like changes to firefox, chrome, chromium, vivaldi, opera triggered this, as it works up to firefox esr 60.x, but does not with later firefox versions. same for chrome-based browsers: older versions work, but not the latest ones.

Keepasshttp seems to fill in user and password, but never click on "OK". Selecting one entry (even if it is only one entry) explicitly makes it fill in user and password and then click "OK" for you.

Browsers not working:

- Vivaldi: 2.7.1628.33 (Stable channel) (64-Bit)

- Opera: 63.0.3368.71

- Chrome: 76.0.3809.132 (Offizieller Build) (64-Bit)

- Firefox: 71.0a1 (2019-09-08) (64-bit)

Browsers working:

- Vivaldi: 2.5 (the first 2.6-versions worked. Later ones not)

- Opera: 60 (61 worked reloading the page. Later ones did not)

- Chrome: 69 (Last version to display a login dialog. Later ones did not)

- Firefox: 60 (68esr wont work any more -- exact version keepasshttp started not showing login dialog any more is unknown)

**Comment by mvenghaus** [comment]
when i start keepass .. it says that the plugin is not compatible and then it is not working .. but okay for me now .. i also switched to keepassxc

**Comment by VivantSenior** [comment]
Doesn't work on KeePass 2.43 + KeePassHttp 1.8.4.2 + Vivaldi 2.7.1628.33

**Comment by kmille** [comment]
Does not work for me on Arch Linux with Keepass 2.42.1 + KeePassHttp 1.8.4.2

"The following plugin is incomppatible with the current KeePass version:
/home/kmille/.local/share/KeePass/PluginCache/<some id>/KeePassHttp.dll"

**Comment by kmille** [comment]
This software is not maintained anymore: https://wiki.archlinux.org/index.php/KeePass#KeePassHTTP_for_Keepass

I solved it by migrating to keepassxc. 
1. It's very easy to migrate
2. keepassxc is actively developed (https://github.com/keepassxreboot/keepassxc)
3. You don't need a plugin for keepassxc

so just 'sudo pacman -S keepassxc'. Then enable the HTTP-API in the settings of keepassxc. Finally get the [Chromium-Plugin](https://chrome.google.com/webstore/detail/keepassxc-browser/oboonakemofpalcgghocfoadofidjkkk?utm_source=chrome-ntp-icon). It looks the same like the keepasshttp-plugin.

[Here is the how to](https://keepassxc.org/docs/keepassxc-browser-migration/)

**Comment by iNoSec** [comment]
Hi all,
keepass worked properly on archlinux until today but now i have the same error message than others 

> "The following plugin is incomppatible with the current KeePass version:
/home/myusername/.local/share/KeePass/PluginCache//KeePassHttp.dll"

**Comment by kmille** [comment]
I migrated to KeepassXC because it's more stable. It also has the funtionality of KeepassHTTP as inbuilt feature.

**Comment by iNoSec2** [comment]
yeah me too and that's work so much better, the KeePassXC-browser work better too
